gatsby-node.js

```diff
@@ -83,6 +85,20 @@ exports.sourceNodes = async () => {
       }
     }
   });
+
+  // Get all MongoDB products for the sidenav
+  const products = await stitchClient.callFunction('fetchAllProducts', [siteMetadata.database]);
+  products.forEach((product) => {
+    createNode({
+      id: createNodeId(`Product-${product.title}`),
+      internal: {
+        contentDigest: createContentDigest(product),
+        type: 'Product',
+      },
+      title: product.title,
+      url: product.baseUrl + product.slug,
+    });
```

Comment:
AFAICT `children` and `parent` are required properties:
```suggestion
      parent: null,
      children: [],
    });
```